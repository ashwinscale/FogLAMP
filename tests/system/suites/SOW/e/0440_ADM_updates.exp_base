FogLAMP started.
Total Messages Transferred: 1
{
  "Good": true,
  "Questionable": false,
  "Substituted": false,
  "Timestamp": "xxx",
  "UnitsAbbreviation": "",
  "Value": 10
}
foglamp x.x.x1
FogLAMP started.
Injection process started.
foglamp x.x.x2
Kills the injection process.
