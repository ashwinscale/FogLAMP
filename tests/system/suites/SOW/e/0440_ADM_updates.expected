DBG - 0440_AD_updates
